extra_scripts/create_clevr_count_data_files.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import json
+import os
+import shutil
+
+from tqdm import tqdm
+
+
+def create_clevr_count_disk_folder(input_path: str, output_path: str):
```

Comment:
is it possible to document this under https://github.com/facebookresearch/vissl/tree/master/vissl/data#vissl-datasets ?

Reply:
Should be fixed in d4e79d89ab08c9caf4f30a0a9157bcc12983b4be.